backend/src/main/java/com/speakfluid/backend/entities/CaptureStep.java

```diff
@@ -64,7 +113,7 @@ public CaptureStep(){
     public void hasNumbers(Message message){
         for(String word : message.getMessage().split(" ")){
             if(word.matches(".*[0-9].*")){
-                scoreAccumulator += 3;
+                scoreAccumulator += 1.0;
```

Comment:
1.0 can be replaced by ScoreStandards.lowMatch

Reply:
Yessir no magic numbers